src/main/java/hudson/plugins/dimensionsscm/DimensionsAPICallback14.java

```diff
@@ -37,9 +38,11 @@ public boolean hasRepositoryBeenUpdated(DimensionsAPI dimensionsAPI, DimensionsC
             if (changeSteps != null) {
                 PathMatcher pathMatcher = dimensionsAPI.getPathMatcher();
                 for (DimensionsChangeStep changeStep : changeSteps) {
+                    String projectPath = PathUtils.normalizeSlashes(changeStep.getProjectPath());
+                    String remotePath = PathUtils.normalizeSlashes(workspace.getRemote());
```

Comment:
I think it's possible for `workspace` to be `null`. What are you actually checking for anyway? Why should checking if the repo has been updated care about the `workspace` in Dimensions CM?

Reply:
This is fix for DEF337415
The same reason as I described for another message.
Just bad variable name... I named it so because method has name .getRemote()..
workspace variable  is taken from those cycle which I have attached as the screen shot to another message (workspace==FilePath object), so it can't be null
FilePath object contains folder from folders list specified by user in Folders field
![image](https://user-images.githubusercontent.com/32028221/72349187-b710db00-36e4-11ea-8336-2259972ec9eb.png)

Reply:
OK - will take a closer look. No need to change the variable name now, but maybe next time we should change these to "monitoredFolder" or something similar 👍